client/app/bundles/course/assessment/submission/suggestionTranslations.ts

```diff
@@ -0,0 +1,56 @@
+import { defineMessages } from 'react-intl';
+
+export const suggestionsTranslations = defineMessages({
+  iAmStuck: {
+    id: 'course.assessment.submission.suggestions.iAmStuck',
+    defaultMessage: 'I am stuck',
+  },
+  myCodeDoesntWork: {
+    id: 'course.assessment.submission.suggestions.myCodeDoesntWork',
+    defaultMessage: "My code doesn't work",
+  },
+  howDoIFixThis: {
+    id: 'course.assessment.submission.suggestions.howDoIFixThis',
+    defaultMessage: 'How do I fix this?',
+  },
+  bestApproach: {
+    id: 'course.assessment.submission.suggestions.bestApproach',
+    defaultMessage: 'Best approach?',
+  },
+  whyIsThisNotWorking: {
+    id: 'course.assessment.submission.suggestions.whyIsThisNotWorking',
+    defaultMessage: 'Why is this not working?',
+  },
+  howToDebug: {
+    id: 'course.assessment.submission.suggestions.howToDebug',
+    defaultMessage: 'How to debug?',
+  },
+  correctSyntax: {
+    id: 'course.assessment.submission.suggestions.correctSyntax',
+    defaultMessage: 'Correct syntax?',
+  },
+  optimizeThisCode: {
+    id: 'course.assessment.submission.suggestions.optimizeThisCode',
+    defaultMessage: 'Optimize this code',
+  },
+  handleThisException: {
+    id: 'course.assessment.submission.suggestions.handleThisException',
+    defaultMessage: 'Handle this exception?',
+  },
+  bestPractice: {
+    id: 'course.assessment.submission.suggestions.bestPractice',
+    defaultMessage: 'Best practice?',
+  },
+  understandThisError: {
+    id: 'course.assessment.submission.suggestions.understandThisError',
+    defaultMessage: 'Understand this error?',
+  },
+  refactorThisCode: {
+    id: 'course.assessment.submission.suggestions.refactorThisCode',
+    defaultMessage: 'Refactor this code?',
+  },
+  reviewMyCode: {
+    id: 'course.assessment.submission.suggestions.reviewMyCode',
+    defaultMessage: 'Review my code?',
+  },
+});
```

Comment:
Also please don't lift wholesale and come up with a more succinct list of translations which convey different intents.
- Handle this exception?
- Understand this error?
- Refactor this code?
- Review my code?
are not questions.

How does "Best approach?" help to convey intent? "Review my approach" is much clearer.

Reply:
truncated